.github/workflows/gh-action.yml

```diff
@@ -0,0 +1,37 @@
+name: Test GitHub Action installer
+on:
+  merge_group:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - install-from-binstall-release.ps1
+      - install-from-binstall-release.sh
+      - action.yml
+  push:
+    branches:
+      - main
+    paths:
+      - install-from-binstall-release.ps1
+      - install-from-binstall-release.sh
+      - action.yml
+
+jobs:
+  test-gha-installer:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ macos-latest, ubuntu-latest, windows-latest ]
+    steps:
+      - name: Install cargo-binstall
+        # this may appear wordy but ensures this test will run successfully on main repo, on forks, and in PRs
+        # e.g. cargo-bins/cargo-binstall@some-pr-branch, or i-love-to-fork/cargo-binstall@my-pr
+        uses: ${{ github.repository }}@${{ github.ref_name }}
```

Comment:
This failed in [CI](https://github.com/cargo-bins/cargo-binstall/actions/runs/5808720649):

> the `uses' attribute must be a path, a Docker image, or owner/repo@ref

Reply:
thanks, I'm testing this in my fork repo https://github.com/smallstepman/cargo-binstall/pulls ; no need to rerun workflows here

Reply:
ok, should be good to go! :)